### PR TITLE
Add task tracking

### PR DESCRIPTION
**Pull request by ryanchard**
This makes the web service record tasks in the database and increment a invocation_counter value in Redis. There is also a new script added to utils to capture usage info each day (to be set as a cron job) such that we can report stats. My main concern with this PR is that it creates a new redis and db connection every time a function is invoked -- which is slow. I've made another issue (#132) to deal with improving that.

### CODE REVIEW

File: models/utils.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from errors import *
 
 
-def create_task(task):
+def create_task(user_id, task_id, function_id, endpoint_id):
```
Comment (ZhuozhaoLi) [inline models/utils.py]: The function name create_task is a bit confusing.

File: utils/store_usage.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    # Combine them together
+    data.update(redis_data)
+    print(data)
+
+    store_data(data, conn, cur)
+    print('done')
+
+
+if __name__ == '__main__':
+    record_usage()
```
Comment (ZhuozhaoLi) [inline utils/store_usage.py]: is it a long running service running on web service?

File: models/utils.py (hunk 1 above)
Comment (ryanchard) [reply models/utils.py]: Changed this to log_invocation

File: utils/store_usage.py (hunk 2 above)
Comment (ryanchard) [reply utils/store_usage.py]: This runs on a crontab. Perhaps we should move it to a scripts directory if we are keeping it in this repo.

**Pull request merged**